source/includes/_install_js_callbacks.md

```diff
@@ -0,0 +1,44 @@
+# Callbacks
+
+```javascript
+wootricSettings = {
+  account_token: 'NPS-XXXXXXX',
+  email: 'nps@example.com',
+  created_at: 1528416000,
+  callbacks: {
+    after_eligible: function(eligible) {
+      if (eligible) {
+        console.log(
+          'A survey will show up in: ' +
+          window.wootric_time_delay + ' seconds'
+        );
+      } else {
+        console.log('A survey will not show up');
+      }
+    },
+    after_show: function() {
+      console.log('The Wootric modal is visible');
+    },
+    after_response: function(score, text) {
+      console.log('A response has been submitted with score:', score, 'and text', text);
+    },
+    after_decline: function() {
+      console.log('A decline has been submitted');
+    }
+  }
+};
+```
+
+The Wootric SDK allows you to use callbacks after certain events happened.
+You can also get information from some of the events through them.
+The list of the events that you can subscribe to goes as follow:
+
+- after_eligible: Runs after the request from Wootric's eligibility server returns.
+                  It can tell you if you were eligible for a survey or not.
+- after_show: Runs after the survey is displayed.
+- after_response: Runs after a answering a survey (or an update on it. I.E. score change).
+                  You can get the score or written feedback out from it.
+- after_decline: Runs after a decline is created.
+
+In order to use them just define them inside the wootricSettings hash as `callbacks` and then
+a new key-value pair containing the name of the callback and its respective function.
```

Comment:
The Wootric SDK allows you to use callbacks after certain events have happened. You can also get information from some of the events. The list of events that you can subscribe to goes as follow:

after_eligible: This callback runs after the request from Wootric's eligibility server returns. It can tell you if you are eligible for a survey or not.

after_show: This callback runs after the survey is displayed.

after_response: This callback runs after the survey is answered. It was also run if the survey is updated. (e.g. score changes) The callback function will receive the score and text.

after_decline: This callback runs after a decline is created.

To use callbacks, define them inside of a `wootricSettings.callbacks` object mapping event to callback function.